masterlist.yaml

```
@@ -26760,3 +26760,19 @@ plugins:
         util: 'SSEEdit v4.0.4c'
       - crc: 0x76F44BFE
         util: 'SSEEdit v4.0.4c'
+        
+  - name: 'RedBag''s Solitude Docks Updated.esp'
```

Some extra whitespace:
```suggestion
        util: 'SSEEdit v4.0.4c'

  - name: 'RedBag''s Solitude Docks Updated.esp'
```